assembl/graphql/schema.py

```diff
@@ -546,7 +546,7 @@ class Post(SecureObjectType, SQLAlchemyObjectType):
     class Meta:
         model = models.Post
         interfaces = (Node, PostInterface)
-        only_fields = ('id', )  # inherits fields from Post interface only
+        only_fields = ('id', 'message_classifier')  # inherits fields from Post interface only
```

Comment:
You can remove message_classifier here. It is said in the comment that it inherits the fields from PostInterface.